src/controllers/SpotifyController.js

```diff
@@ -10,24 +10,43 @@ var config = {
   previousSelector: '#previous'
 }
 
-if (document.querySelector('#app-player')) { // Old Player
-  config.artworkImageSelector = '#cover-art .sp-image-img';
-  config.frameSelector = '#app-player';
-  config.playStateSelector = '#play-pause';
-  config.playPauseSelector = '#play-pause';
-  config.titleSelector = '#track-name';
-  config.artistSelector = '#track-artist';
-} else { // New Player
-  config.artworkImageSelector = '#large-cover-image';
-  config.frameSelector = '#main';
-  config.playStateSelector = '#play';
-  config.playPauseSelector = '#pause';
-  config.titleSelector = '.caption .track';
-  config.artistSelector = '.caption .artist';
+// New version of spotify
+if (window.location.hostname === 'open.spotify.com') {
```

Comment:
I think this is fine for now, don't want to mess with the mapping code at the moment.